docs-preview/src/extension.ts

```diff
@@ -62,3 +66,11 @@ export function codeSnippets(md, options) {
     };
     md.core.ruler.before("normalize", "codesnippet", importCodeSnippet);
 }
+
+export function sendTelemetryData(commandOption: string) {
```

Comment:
@jamarw - Everything looks good! One thought... could this function be moved under common.ts?

Reply:
@adunndevster Great idea.  Maybe I should create a single function in the other extensions and reduce a few lines of code.